README.md

```diff
@@ -116,11 +115,11 @@ $ heroku buildpacks -a yourappname
 - Go to the [Google Developers Console](https://console.developers.google.com/project) and create a new project
 - Once you've created the project, go to `APIs` and add the `Contacts API` and the `Google+ API` (you won't need a Google+ account to sign in, this is just an annoying Google quirk).
 - Then go to `Credentials` and `Create a new Client ID`. You'll need the app's production URL to complete this step so if you're using the Heroku button, do that first. You can use your production URL for the `JavaScript Origins` setting, but make sure to use `http://yourdomain.com/auth/google_oauth2/callback` for in the `Redirect URIs` setting. It's a good idea to also add the same URL but with the HTTPS protocol to ensure that if you ever force SSL, Google will still accept the redirect.
-- Don't forget to go update the `GOOGLE_KEY` and `GOOGLE_SECRET` environment variables with the credentials Google gave you when you created your Client ID, otherwise the redirection process will fail.
+- Don't forget to go update the `OAUTH_PROVIDER_KEY` and `OAUTH_PROVIDER_SECRET` environment variables with the credentials Google gave you when you created your Client ID, otherwise the redirection process will fail.
```

Comment:
"... with the credentials **Google** gave you...". Should be "the provider" gave you.